src/components/ProgressCircle/ProgressCircle.test.tsx

```
@@ -0,0 +1,462 @@
+
+    it('handles -Infinity value', () => {
+      render(<ProgressCircle value={-Infinity} showPercentage />);
+      const percentage = screen.getByText('0%');
+      expect(percentage).toBeInTheDocument();
+    });
+
+    it('handles empty string as label', () => {
+      const { container } = render(<ProgressCircle value={50} label="" />);
+
+      // When label is empty string, it still renders but with empty content
+      const allSpans = container.querySelectorAll('span');
```

ESLint

[error] 'allSpans' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)